theano/sandbox/cuda/dnn.py

```diff
@@ -368,6 +368,27 @@ def ensure_float(val, default, name):
     return val
 
 
+def do_merge_gpu_alloc_empty(node1, node2):
```

Comment:
The check for the op equility is already done in MergeFeature. But I agree, it would be better to do it there too.